cmd/cert-checker/main.go

```diff
@@ -140,23 +139,23 @@ func newChecker(saDbMap certDB,
 	}
 }
 
-func (c *certChecker) getCerts(ctx context.Context, unexpiredOnly bool) error {
-	c.issuedReport.end = c.clock.Now()
-	c.issuedReport.begin = c.issuedReport.end.Add(-c.checkPeriod)
-
-	args := map[string]interface{}{"issued": c.issuedReport.begin, "now": 0}
-	if unexpiredOnly {
-		args["now"] = c.clock.Now()
-	}
+func (c *certChecker) getCerts(ctx context.Context) error {
+	c.issuedReport.end = c.clock.Now().Truncate(time.Second).Add(time.Second)
```

Comment:
My thinking here was that for "end" we should round up and for "begin" we should round down. Since there's no "round up" function for Duration, the first one rounds down and then adds a second. I'll add a comment.